src/pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -125,6 +124,17 @@ def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
             self.scaler.load_state_dict(state_dict)
 
 
+class NativeMixedPrecisionPlugin(MixedPrecisionPlugin):
```

Comment:
Follow-up items (out of scope for this PR):

- Remove native_ from the filename
- Remove "native" from the FSDP classes